### ISSUE TITLE
Analyze phenotypes: source ontology interface and taxon configuration

### ISSUE THREAD

**Issue by jmcmurry** [monarch-initiative/monarch-legacy#1293]
We currently have confusing scenarios like this one we need species-specific configuration.

<img width="241" alt="screen shot 2016-07-11 at 9 32 48 am" src="https://cloud.githubusercontent.com/assets/3658863/16738631/5f5718e8-474b-11e6-8e77-c5890a2be4e5.png">

1) Is there any scenario in which there would be more than one species in the phenotype profile? I can't think of one. 
2) Are there ever, for a single input, duplicate terms matched from different ontologies? (Eg. MESH and HPO) I haven't seen any, so am going to consider this scenario remote enough to not worry about. Eg. that our clique merging is working as it should.

In either unlikely scenario 1 or 2, we would also need an indication of source ontology to be present in the autocomplete matches themselves. For not let's not worry about it. Feel free to push back on this if my assessment is hasty.

**Comment by harryhoch** [comment]
Agreed that we should indicate source and limit to one set of terms. I think this has been discussed and needs a bit of ontology work along with UI work.

Might want to think about this in terms of a general redesign on the UI side, if we want to add support for ontology navigation as well.

**Comment by kshefchek** [comment]
It looks like Lewy Bodies from HP and MP are not equivalent in SciGraph, I would check UberPheno to confirm.
